app/views/service_providers/_form.html.erb

```diff
@@ -350,7 +350,9 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
 
   <fieldset class="usa-fieldset-inputs custom-help-text">
     <legend>Custom help text</legend>
-    <p>You can specify help text in these different areas to alert users of important information for logging into your app</p>
+    <p>You can specify help text in these different areas to alert users of important information for logging into your app. Permitted HTML elements include the following:
+    <%= ServiceProvider::ALLOWED_HELP_TEXT_HTML_TAGS.map { |t| "<code>#{t}</code>" }.join(", ") %>.
```

Comment:
Looks like it's escaped:

![image](https://user-images.githubusercontent.com/1779930/160396332-b7ebdafc-c60c-48c2-8e7f-d828ce13ac5e.png)

Reply:
🤦 thank you for checking. I'm guessing I just need to throw an `html_safe` at the end

Reply:
done and checked locally that it looks ok. Thanks again @aduth !